SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/SpatialRPCTimeoutMap.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/SpatialRPCTimeoutMap.h"
+#include "GameFramework/PlayerStart.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRPCTimeout/SpatialTestRPCTimeout.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRPCTimeout/SpatialTestRPCTimeoutGameMode.h"
+
+USpatialRPCTimeoutMap::USpatialRPCTimeoutMap()
+	: UGeneratedTestMap(EMapCategory::CI_PREMERGE, TEXT("SpatialRPCTimeoutMap"))
+{
+	// clang-format off
+	SetCustomConfig(TEXT("[/Script/SpatialGDK.SpatialGDKSettings]") LINE_TERMINATOR
+		TEXT("QueuedIncomingRPCWaitTime=0") LINE_TERMINATOR
+		TEXT("[/Script/UnrealEd.LevelEditorPlaySettings]") LINE_TERMINATOR
+		TEXT("PlayNumberOfClients=2"));
```

Comment:
I think this can be replaced with SetNumberOfClients?

Reply:
Yeah it can be, will fix soon.

Though the vast majority of the functional test still uses this manual config strings, maybe we should think about replacing it accross the board